lib/Jsrt/Core/JsrtContextCore.h

```diff
@@ -163,7 +163,6 @@ class ChakraCoreHostScriptContext sealed : public HostScriptContext
 
     HRESULT EnqueuePromiseTask(Js::Var taskVar) override
     {
-        AssertMsg(false, "jsrt should have set the promise callback");
```

Comment:
I took this out after speaking with Jimmy, because we wanted to add a unit test for this and he said testing asserts was problematic.